dds2val/src/make_executable.sh

```diff
@@ -0,0 +1,3 @@
+if [[ "$TARGETARCH" == "arm64" ]]; then 
+  pyinstaller --clean --paths=. --add-binary="/opt/venv/lib/python3.9/site-packages/cyclonedds.libs/libddsc-62ba48e6.so.0.11.0:cyclonedds.libs/" --add-data="mapping.yml:." -F -s ddsfeeder.py; else pyinstaller --clean --paths=. --add-data="mapping.yml:." -F -s ddsfeeder.py 
```

Comment:
Risky as we do not specify which libddsc version we will have?

In [dbcfeeder](https://github.com/eclipse/kuksa.val.feeders/blob/main/dbc2val/Dockerfile#L37) I used a more generic solution to collect data. Would possibly `--collect-binaries cyclonedds` work?

Reply:
Problem here is it is collected by pyinstaller with this but DDS searches in a folder for it. But this is not ported by pyinstaller. Will ivestigate further.